### PR TITLE
views: create a home page view

### PR DESCRIPTION
**Pull request by tfidfwastaken**
Create a homepage view with basic styling and non-functional buttons.
Also added a favicon.

Play with it here: https://bugle-forms-staging.herokuapp.com/

Closes: https://app.shortcut.com/nilenso/story/124/as-a-user-i-want-to-see-the-homepage-so-that-i-can-interact-with-the-web-app

### CODE REVIEW

File: resources/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (kirang89) [inline resources/site.webmanifest:1]: question: What is the purpose of this file?

Comment (tfidfwastaken) [reply resources/site.webmanifest:1]: This information is used by platforms that allow you to pin web pages as "apps" on their screen. This was autogenerated by the favicon creator I used.

Comment (kirang89) [reply resources/site.webmanifest:1]: oh hmm TIL.

File: src/bugle_forms/core.clj [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
-(defn hello-world [request]
-  {:status 200
-   :headers {"Content-Type" "text/html"}
-   :body
-   "<iframe src='//commons.wikimedia.org/wiki/File:FirstCall.ogg?embedplayer=yes'
-   width='300' height='20' frameborder='0' webkitAllowFullScreen mozallowfullscreen allowFullScreen>
-   </iframe>"})
+(defn home [request]
+  (response/response
+   (html5 (layout/application "Bugle Forms" home/home-content))))
```
Comment (kirang89) [inline src/bugle_forms/core.clj]: suggestion (nitpick): In general, it's a good idea to give assign literal strings or numbers to a var and give it a meaninful name that conveys intent.

File: src/bugle_forms/views/home.clj [hunk 3]
```diff
@@ -0,0 +1,10 @@
+(ns bugle-forms.views.home)
+
+(def home-content
```
Comment (kirang89) [inline src/bugle_forms/views/home.clj]: suggestion (non-blocking): The `home` in the var name is redundant considering the namespace you've defined it it. It's cleaner to name functions to omit redundancy when calling with the namespace i.e `bugle-forms.views.home/content`

File: src/bugle_forms/views/layout.clj [hunk 4]
```diff
@@ -0,0 +1,24 @@
+(ns bugle-forms.views.layout
+  (:require [hiccup.page :refer [include-css]]))
+
+(defn- navbar [nav-items]
+  [:div {:class "nav"}
+   [:a {:href "/" :title "home"} "🎺"]
+   (map (fn [[link-title link-url]]
+          [:a {:href link-url :title link-title} link-title]) nav-items)])
```
Comment (kirang89) [inline src/bugle_forms/views/layout.clj]: suggestion: Aligning function args (for `map` in this case) vertically makes it easier to read.
(https://guide.clojure.style/#vertically-align-fn-args)

**Review by kirang89 — APPROVED** [review]
I've left some minor comments, but the PR looks good to merge.

**Pull request merged**